client/src/services/fileOperationManager.ts

```diff
@@ -51,6 +51,12 @@ enum FileOperationState {
   EntryCopied,
 }
 
+enum FileOperationTabs {
```

Comment:
Nop. `tabs` are how we chose to display it, it has nothing to do with the operation manager. It doesn't care what we do with the information it provides.